### ISSUE TITLE
[Bug]: Does not work with --experimental_sibling_repository_layout

### ISSUE THREAD

**Issue by matthewjh** [aspect-build/rules_swc#285]
### What happened?

When building an SWC output with `--experimental_sibling_repository_layout`, SWCCompile exits with an error:

```
src/main/tools/linux-sandbox-pid1.cc:548: "execvp(external/swc_linux-x64-gnu/swc-linux-x64-gnu, 0x14e1720)": No such file or directory  
```

### Version

Development (host) and target OS/architectures:

Ubuntu 22, linux amd64

Output of `bazel --version`:

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:

rules_swc 2.0.0

Language(s) and/or frameworks involved:
7.4.1

**Comment by jbedard** [comment]
FWIW I think almost every aspect ruleset using toolchains has this issue where we prepend `external/` to the path of the binary, rules_esbuild being another example (which has the path prepended so I assume it has the bug).

**Comment by alexeagle** [comment]
@matthewjh do you have any idea the severity of this issue - meaning, have you seen some indication this flag is going to be flipped and users will be broken unless we take action?
Another way to ask: what led you to try that flag?

**Comment by matthewjh** [comment]
@alexeagle I hit this issue, due an external directory in our source tree:

https://github.com/bazelbuild/bazel/issues/4508

**Comment by alexeagle** [comment]
It seems like this flag isn't likely to be flipped https://github.com/bazelbuild/bazel/discussions/20500#discussioncomment-10623863
so I don't think we'd bother being compliant with its constraints.
